CONTRIBUTING.md

```diff
@@ -38,7 +38,7 @@ not already exist.
 * Follow the coding style in use.
 * Check for unnecessary whitespace with `git diff --check` before committing.
 * Make sure you have added the necessary tests for your changes.
-* Run _all_ the tests to assure nothing else was accidentally broken.
+* Run _all_ the tests via `$grunt test`command to assure nothing else was accidentally broken.
```

Comment:
suggestion (dunno whether 'bug' or 'issue' is best word below):
Run _all_ the tests using the `$grunt test`command to make sure your change did not accidentally introduce a bug.

Reply:
👍 Tks Done.